### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: pytest_tests/tests/services/rest_gate/test_rest_gate.py [hunk 1]
```diff
@@ -7,19 +7,19 @@
...
     try_to_get_object_and_expect_error,
     upload_via_rest_gate,
-    quote,
 )
-from helpers.neofs_verbs import put_object_to_random_node
 from helpers.utility import wait_for_gc_pass_on_storage_nodes
 from helpers.wellknown_acl import PUBLIC_ACL
-from rest_gw.rest_utils import get_object_and_verify_hashes
 from neofs_env.neofs_env_test_base import NeofsEnvTestBase
+from rest_gw.rest_utils import get_object_and_verify_hashes
```
Comment (roman-khimov) [inline pytest_tests/tests/services/rest_gate/test_rest_gate.py:22]: Formatting change, not related to marks.

File: pytest_tests/tests/services/rest_gate/test_rest_gate_new.py [hunk 2]
```diff
@@ -8,19 +8,19 @@
...
     new_upload_via_rest_gate,
-    get_epoch_duration_via_rest_gate,
+    try_to_get_object_and_expect_error,
 )
-from helpers.neofs_verbs import put_object_to_random_node
 from helpers.utility import wait_for_gc_pass_on_storage_nodes
 from helpers.wellknown_acl import PUBLIC_ACL
-from rest_gw.rest_utils import get_object_and_verify_hashes
 from neofs_env.neofs_env_test_base import NeofsEnvTestBase
+from rest_gw.rest_utils import get_object_and_verify_hashes
```
Comment (roman-khimov) [inline pytest_tests/tests/services/rest_gate/test_rest_gate_new.py:23]: Here as well.

**Review by roman-khimov — COMMENTED** [review]
How come 5f5f72a313520452fc46fff5f7705accffc64aba commits a `warp` binary?

Can e have a bit more details for 5de9dabcba19070a0798ca6cb1e9531bff161a28?

Where are references to original issue?

File: pytest_tests/tests/services/rest_gate/test_rest_gate.py (hunk 1 above)
Comment (evgeniiz321) [reply pytest_tests/tests/services/rest_gate/test_rest_gate.py]: removed

File: pytest_tests/tests/services/rest_gate/test_rest_gate_new.py (hunk 2 above)
Comment (evgeniiz321) [reply pytest_tests/tests/services/rest_gate/test_rest_gate_new.py]: removed

**Comment by evgeniiz321** [comment]
> How come [5f5f72a](https://github.com/nspcc-dev/neofs-testcases/commit/5f5f72a313520452fc46fff5f7705accffc64aba) commits a `warp` binary?
> 
> Can e have a bit more details for [5de9dab](https://github.com/nspcc-dev/neofs-testcases/commit/5de9dabcba19070a0798ca6cb1e9531bff161a28)?
> 
> Where are references to original issue?

Removed warp binary, was there by accident.
Added a little bit more details to the commit with an invalid escape sequence.
Added a reference to the new issue with warnings.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**